develop/blocks/first-block.md

```diff
@@ -151,6 +151,8 @@ This example adds the "Condensed Dirt" block to the `shovel` tag.
 
 @[code](@/reference/latest/src/main/resources/data/minecraft/tags/mineable/shovel.json)
 
+If you wish to have your block be required to be mined by a tool, you'll want to append `.requiresTool()` to your block settings.
+
 ## Mining Levels {#mining-levels}
```

Comment:
What's the difference between `.requiresTool()` and using tags?

Reply:
Using tags and using `.requiresTool()` is necessary in order to prevent a block loot table from dropping with an incorrect tool. Without the `requiresTool()`, the block loot table will always drop, regardless of tags.